streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/trait/FlinkClientTrait.scala

```diff
@@ -102,7 +102,11 @@ trait FlinkClientTrait extends Logger {
         }
       case _ =>
         if (submitRequest.userJarFile != null) {
-          val uri = PackagedProgramUtils.resolveURI(submitRequest.userJarFile.getAbsolutePath)
+          var jarPath = submitRequest.userJarFile.getAbsolutePath
```

Comment:
Thanks very much for your contribution. It is not recommended to make changes to the code here. You can make changes to the path in the ApplicationService as follows: 

before:
<img width="835" alt="image" src="https://github.com/apache/incubator-streampark/assets/13284744/fdf040f2-af53-407b-aec1-6ac3e5845494">

after:
<img width="912" alt="image" src="https://github.com/apache/incubator-streampark/assets/13284744/87c40968-b035-4a40-890d-28446358a0f0">

You can do a global search to improve all the path concat in this way. Looking forward to your updates! 🌟

Reply:
> Thanks very much for your contribution. It is not recommended to make changes to the code here. You can make changes to the path in the ApplicationService as follows:
> 
> before: <img alt="image" width="835" src="https://private-user-images.githubusercontent.com/13284744/306326764-fdf040f2-af53-407b-aec1-6ac3e5845494.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMjQ1OTYsIm5iZiI6MTcxMDEyNDI5NiwicGF0aCI6Ii8xMzI4NDc0NC8zMDYzMjY3NjQtZmRmMDQwZjItYWY1My00MDdiLWFlYzEtNmFjM2U1ODQ1NDk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzExVDAyMzEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4NWVkMGJmNDU1NjAyNDUyOTA5OGZmYjFkOGI1MTJjNmUxZDRiMDFjOGU5MTUzODM0NWQ1MDNlOTFjMDdlNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xcSO_lBglKmb4UFjChMmwaKbAdLdaG1hvyBEj0D4R0Y">
> 
> after: <img alt="image" width="912" src="https://private-user-images.githubusercontent.com/13284744/306325745-87c40968-b035-4a40-890d-28446358a0f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMjQ1OTYsIm5iZiI6MTcxMDEyNDI5NiwicGF0aCI6Ii8xMzI4NDc0NC8zMDYzMjU3NDUtODdjNDA5NjgtYjAzNS00YTQwLTg5MGQtMjg0NDYzNThhMGYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzExVDAyMzEzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM0MGQyYzg5MDEwZjA1YzhhZGRiNTZkNDRlYjdmMmQ2ZmMwM2I1OWIwMjA2NjQ5OGIxNjdiOWQ2OGQ1Y2E5ZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.C-oWGJvTmsbwaq93qzhlXiBx-X0c-TFGiHeg2ystMAw">
> 
> You can do a global search to improve all the path concat in this way. Looking forward to your updates! 🌟

hi~ @wolfboys , tks for you advice. Can you successfully test it locally? I attempted to debug it in this manner locally but failed, and it's still showing the same error as before.